src/FiberMap.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace React\Async;
+
+use Fiber;
```

Comment:
nit: exclude imports for global classes and use `\Fiber` below instead for consistency?

Reply:
Personally I prefer importing them, but will change it for consistency sake